apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -524,6 +566,76 @@ export function buildMachine(
                   'poll_worker_auth_ended_unexpectedly',
               },
             },
+
+            loading_new_sheet: {
+              invoke: [pollPaperStatus(), pollAuthStatus()],
+              entry: (context) => context.driver.loadPaper(),
+              on: {
+                PAPER_INSIDE_NO_JAM: 'validating_new_sheet',
```

Comment:
Probably a question I should have left on the Figma file, but is there no way around adding this initial scan + interpret on paper load? is the logic since we've opened the possibility of ballot re-insertion, we need to scan and interpret all sheets that are loaded, even if we have high confidence they're blank (such as when loaded from the poll worker page for a new session)?

Reply:
One of the product requirements was to allow insertion of ballots at the start of the session as well (that covers cases where a voting session might've ended before the voter had a chance to re-insert a pulled ballot).

Now that we've decided to add a `"Cast a Previously Printed Ballot"` section to the poll worker screen, we could potentially have separate paths for inserting a printed vs blank sheet if we need to optimise the mainline path - combining does reduce the chance of use error.

And I think we had the potential issue of potentially non-blank sheets being inserted before explicitly enabling it with this project - this change doesn't fully address that, since the current interpreter implementation returns a `BlankPage` for any non-blank, non-BMD-ballot sheet.

Reply:
Makes sense. Without data on the initial scan + interpret delay, we don't need to optimize for it now.